go/libkb/pgp_key.go

```diff
@@ -638,14 +638,21 @@ func (k *PGPKeyBundle) SignToString(msg []byte) (sig string, id keybase1.SigID,
 
 func (k PGPKeyBundle) VerifyStringAndExtract(sig string) (msg []byte, id keybase1.SigID, err error) {
 	var ps *ParsedSig
+
 	if ps, err = PGPOpenSig(sig); err != nil {
-		return
-	} else if err = ps.Verify(k); err != nil {
-		return
+		return msg, id, err
```

Comment:
Personal preference: If we got rid of the named returns, this would turn into `return nil, "", err`, which is a little clearer maybe?